cache-config/t3c-check-refs/config/config.go

```diff
@@ -106,13 +112,12 @@ func InitConfig(appVersion string, gitRevision string) (Cfg, error) {
 		return Cfg{}, errors.New("Too many verbose options. The maximum log verbosity level is 2 (-vv or --verbose=2) for errors (0), warnings (1), and info (2)")
 	}
 
+	filesAddingInput := false
 	filesAddingSet := map[string]struct{}{}
-	for _, fileAdding := range strings.Split(*filesAdding, ",") {
-		fileAdding := strings.TrimSpace(fileAdding)
-		if fileAdding == "" {
-			continue
-		}
-		filesAddingSet[fileAdding] = struct{}{}
+	if strings.ToLower(strings.TrimSpace(*filesAdding)) == ArgFilesAddingInput {
```

Comment:
Why `strings.ToLower()`?

Reply:
Just a safety. There's no reason not to accept e.g. `INPUT` or `Input`